src/Nri/Ui/Modal/V5.elm

```diff
@@ -1,17 +1,59 @@
 module Nri.Ui.Modal.V5 exposing
-    ( info, warning
-    , Model, init, Msg, update, subscriptions
+    ( Model, init, Msg, update, subscriptions
+    , info, warning
     , viewContent, viewFooter
     , launchButton, closeButton
     , primaryButton, secondaryButton, dangerButton
     )
 
 {-| Changes from V4:
 
-@docs info, warning
+  - Remove dependence on Assets
+  - Adds keyboard support (escape key to exit, tabs contained within the modal)
+
+These changes have required major API changes. Be sure to wire up subscriptions!
```

Comment:
Hmm, is there some we we could check whether subscriptions were wired up?   I can't think of an easy way to do that.

Reply:
Yeah, it seems really easy to miss! I'm not sure